internal/app/singularity/actions.go

```diff
@@ -310,6 +310,14 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 	engineConfig := singularity.NewConfig()
 
 	configurationFile := buildcfg.SYSCONFDIR + "/singularity/singularity.conf"
+	configFile, err := os.Stat(configurationFile)
+	if err != nil {
+		sylog.Fatalf("Error with Stat() on singularity.conf: %v", err)
+	}
+	if configFile.Sys().(*syscall.Stat_t).Uid != 0 {
```

Comment:
Should we check group too ?

Reply:
Only ownership was checked in 2.x, and we're replicating that.  `src/util/suid.c` ~line 75

Reply:
Well, just because we were only checking user ownership in 2.x doesn't mean we should only check the same here.  Seems sensible to check the group.  And probably not hard either right?

Reply:
As I told @ArangoGutierrez , hard-coding a group check for root will not make you fans with folks that change the perms like:
```
chmod 664 $sysconfdir/singularity/singularity.conf
chown root:singularity $sysconfdir/singularity/singularity.conf
```
So, the _group_ **singularity** can edit the `singularity.conf` file. Ownership of the file fine, but not group.